src/js/components/search/visualizations/rank/RankVisualizationScopeButton.jsx

```diff
@@ -34,13 +34,20 @@ export default class RankVisualizationScopeButton extends React.Component {
             description += ' (currently selected)';
         }
 
+        let clickEvent = '';
+        if (this.props.label == 'Offices') {
+            clickEvent = null;
+        } else {
+            clickEvent = this.clickedButton;
+        }
+
         return (
             <button
                 className={`scope-button${activeClass}`}
                 value={this.props.value}
                 title={description}
                 aria-label={description}
-                onClick={this.clickedButton}>
+                onClick={clickEvent}>
```

Comment:
Instead of variably nulling the `onClick` event, you should set the `disabled` attribute to `true`. This prevents the click event from occurring and also informs screen readers for visually impaired users that the button is inoperable.